### PR TITLE
Web SDK Support 

### PR DESCRIPTION
**Pull request by KambleSonam**
- Bindings and plugin file added
- Custom Widget Added for web inbox

### CODE REVIEW

File: .metadata
Comment (darshanclevertap) [inline .metadata:1]: @KambleSonam Do we need this file? Or can we add this to `.gitignore` ?

File: example/lib/main.dart [hunk 1]
```diff
@@ -53,6 +58,24 @@ class _MyAppState extends State<MyApp> {
     initPlatformState();
     activateCleverTapFlutterPluginHandlers();
     CleverTapPlugin.setDebugLevel(3);
+    if (kIsWeb) {
+      CleverTapPlugin.init("WRK-485-456Z", "sk1-staging-4", "wzrkt.com");
```
Comment (darshanclevertap) [inline example/lib/main.dart:62]: @KambleSonam Remove the Account ID and Region and change it to `YOUR_ACCOUNT_ID` and `YOUR_ACCOUNT_REGION`

File: lib/clevertap_plugin.dart [hunk 2]
```diff
@@ -39,26 +40,31 @@ class CleverTapPlugin {
...
+      const MethodChannel('clevertap_plugin/native_to_dart');
 
   static final CleverTapPlugin _clevertapPlugin =
       new CleverTapPlugin._internal();
 
   factory CleverTapPlugin() => _clevertapPlugin;
 
   static const libName = 'Flutter';
+
   static const libVersion = 10901; // If the current version is X.X.X then pass as X0X0X
```
Comment (darshanclevertap) [inline lib/clevertap_plugin.dart:62]: @KambleSonam @shivamsharma2710 I think we should make the Flutter release v2.0.0 or `20000`

File: pubspec.yaml
Comment (darshanclevertap) [inline pubspec.yaml:1]: @KambleSonam @shivamsharma2710 I think we should update the Flutter SDK version to 2.0.0 or `20000` to mark the Flutter Web support

**Review by darshanclevertap — APPROVED** [review]

File: lib/clevertap_plugin.dart [hunk 3]
```diff
@@ -39,26 +40,31 @@ class CleverTapPlugin {
...
       cleverTapOnVariablesChangedHandlers = [];
-  static List<CleverTapOnValueChangedHandler>
-      cleverTapOnValueChangedHandlers = [];
+  static List<CleverTapOnValueChangedHandler> cleverTapOnValueChangedHandlers =
+      [];
 
-  static const MethodChannel _dartToNativeMethodChannel = const MethodChannel('clevertap_plugin/dart_to_native');
-  static const MethodChannel _nativeToDartMethodChannel = const MethodChannel('clevertap_plugin/native_to_dart');
+  static const MethodChannel _dartToNativeMethodChannel = const MethodChannel(
+      'clevertap_plugin/dart_to_native', const StandardMethodCodec());
```
Comment (shivamsharma2710) [inline lib/clevertap_plugin.dart:51]: Do we need `StandardMethodCodec()` for dart to web communication?

File: .metadata
Comment (KambleSonam) [reply .metadata:1]: No @darshanclevertap . Adding it to `.gitignore`

File: lib/clevertap_plugin.dart (hunk 3 above)
Comment (KambleSonam) [reply lib/clevertap_plugin.dart]: Removed `StandardMethodCodec()`

File: pubspec.yaml
Comment (KambleSonam) [reply pubspec.yaml:1]: Updated the version in all required files

**Review by darshanclevertap — APPROVED** [review]

**Review by shivamsharma2710 — APPROVED** [review]
LGTM

**Pull request merged**